### PR TITLE
Replace in-line style guidelines with link

### PR DESCRIPTION
**Pull request by croaky**
- Use shorter sentences to describe each step.
- Begin document with h1 header to pass Markdown linting.

https://github.com/thoughtbot/templates/issues/4

### CODE REVIEW

File: CONTRIBUTING.md.template [hunk 1]
```diff
@@ -1,53 +1,31 @@
...
+[commit]: http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html
 
-* Two spaces, no tabs.
-* No trailing whitespace. Blank lines should not have any space.
-* Prefer &&/|| over and/or.
-* MyClass.my_method(my_arg) not my_method( my_arg ) or my_method my_arg.
-* a = b and not a=b.
-* Follow the conventions you see used in the source already.
-)
+Wait for feedback. Discuss the feedback. Make any necessary changes.
```
Comment (mike-burns) [inline CONTRIBUTING.md.template]: I actively dislike reading short, choppy sentences like this (it is the reason I can't read Chuck Palahniuk). They are technically correct sentences, so there's no need to change them for me, but I'd prefer:

> Others will give constructive feedback. This is a time for discussion and improvements, and making the necessary changes will be required before we can merge the contribution.

**Comment by tute** [comment]
I also prefer more prose-like form, but this is a step forward for me.

**Comment by mike-burns** [comment]
I do agree with @tute - this is a great improvement.

**Comment by croaky** [comment]
Prose-ified. Other suggestions?

**Comment by tute** [comment]
LGreatTM!

**Comment by mike-burns** [comment]
<3

**Pull request merged**